### PR TITLE
Styling (Issue #17): Make Add Item View look welcoming

### LINKED ISSUE #17: 17. As a user, I want the add an item screen to look pretty so the app looks professional and welcoming.
AC:

* The add an item screen (the screen a user uses to add a new item to their list) looks something like the following (or equivalent, use your creativity!):

add-item

### PR DESCRIPTION
**Pull request by brewswain**
## Description

Updated  `<AddItem />` view to be more attractively styled and responsive.

## Related Issue
 Closes issue #17 

## Acceptance Criteria

- The add an item screen (the screen a user uses to add a new item to their list) looks something like the following (or equivalent, use your creativity!):

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :hammer: Refactoring       |

### Before

image

### After

image

## Testing Steps / QA Criteria

- From your terminal, run `git checkout bl-sd-additem-styling`.
- Once on the ` bl-sd-additem-styling` branch, run `npm run start`.
- Create a new shopping list or access an existing one (if you already have one).
- Click the "Add Item" button to navigate to the `AddItem` view.

### CODE REVIEW

File: src/components/AddItem/AddItem.style.scss [hunk 1]
```diff
@@ -62,3 +86,99 @@
...
+
+  // modal styling
+  .list {
+    &__card {
+      max-width: 90vw;
+    }
+  }
+}
+
+// iPhone 5/SE specific layout
```
Comment (joeylaguna) [inline src/components/AddItem/AddItem.style.scss:171]: As a former SE user for many years, I really appreciate seeing this!

**Comment by joeylaguna** [comment]
Wow! This looks so nice! It's like a totally different app 😄 . I love that you designed for iPhone 5/SE as well, very nice touch! Approved.

**Review by joeylaguna — APPROVED** [review]

File: src/components/AddItem/AddItem.style.scss (hunk 1 above)
Comment (brewswain) [reply src/components/AddItem/AddItem.style.scss:171]: Hahaha, dude this bit had me being like "...do i REALLY got to do this rn"

**Comment by brewswain** [comment]
> Wow! This looks so nice! It's like a totally different app 😄 . I love that you designed for iPhone 5/SE as well, very nice touch! Approved.

Tysm! im excited to see it entirely come together

File: src/components/AddItem/AddItem.component.jsx [hunk 2]
```diff
@@ -84,38 +106,81 @@ const AddItem = props => {
...
-        <div className="item__form__wrapper">
-          <div className="tooltip__container">
-            {/*conditionally renders tooltip based on if isAdded is truthy or falsy */}
-            <div
-              className={`${
-                isAdded ? 'tooltip--visible' : 'tooltip--invisible'
-              } tooltip`}
-            >
-              Success! Item added.
+    <div className="page__container">
```
Comment (CodeMeKathy) [inline src/components/AddItem/AddItem.component.jsx:122]: I love this feature; great idea!

File: src/components/Footer/Footer.component.jsx [hunk 3]
```diff
@@ -13,6 +13,15 @@ const Footer = () => (
     >
       Home
     </NavLink>
+    {addItem ? (
+      <NavLink
```
Comment (CodeMeKathy) [inline src/components/Footer/Footer.component.jsx:17]: Thanks for adding the List link back into the footer. Super helpful!

**Review by CodeMeKathy — APPROVED** [review]
Everything looks terrific and cohesive.  Great work!

**Review by daredtech — CHANGES_REQUESTED** [review]
Is there a reason why `Add Item` no longer has a list above? And `List` inside the footer? 

Screen Shot 2020-09-27 at 11 45 21 AM

**Pull request merged**